src/main/java/uk/gov/hmcts/reform/divorce/orchestration/controller/CaseController.java

```diff
@@ -45,20 +44,20 @@
         @ApiResponse(code = 400, message = "Bad Request")})
     public ResponseEntity<CaseCreationResponse> submitCase(
         @RequestHeader(value = AUTHORIZATION_HEADER) String authorizationToken,
-        @RequestBody @ApiParam("Divorce Session") Map<String, Object> payload) throws WorkflowException {
+        @RequestBody @ApiParam("Divorce Session") Map<String, Object> payload) {
 
-        Map<String, Object> serviceResponse = caseService.submitDraftCase(payload, authorizationToken);
+        try {
+            Map<String, Object> serviceResponse = caseService.submitDraftCase(payload, authorizationToken);
 
-        if (serviceResponse.containsKey(VALIDATION_ERROR_KEY)) {
-            log.error("Bad request. Found this validation error: {}", serviceResponse.get(VALIDATION_ERROR_KEY));
-            return ResponseEntity.status(HttpStatus.BAD_REQUEST).build();
+            CaseCreationResponse caseCreationResponse = new CaseCreationResponse();
+            caseCreationResponse.setCaseId(String.valueOf(serviceResponse.get(ID)));
+            caseCreationResponse.setStatus(SUCCESS_STATUS);
+            return ResponseEntity.ok(caseCreationResponse);
+        } catch (CaseAlreadyExistsException caseAlreadyExistsException) {
+            log.error(caseAlreadyExistsException.getMessage(), caseAlreadyExistsException);
+            return ResponseEntity.badRequest().build();
         }
```

Comment:
you could move this to global exception handler which we have ? So we won't have any try catch block here